go.mod

```diff
@@ -53,6 +54,7 @@ require (
 	golang.org/x/oauth2 v0.3.0
 	gopkg.in/resty.v1 v1.12.0
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
do you know if it would be possible to use only one version of this dependency?

Reply:
+1 on the idea but I've not investigated deep, I just did a `go mod tidy` - this dep was already existing https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1504/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L64

Reply:
This is taken care of in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1463